pkg/daemon/daemon.go

```diff
@@ -1004,7 +1004,10 @@ func (dn *Daemon) checkStateOnFirstRun() error {
 		if !osMatch {
 			glog.Infof("Bootstrap pivot required to: %s", targetOSImageURL)
 			// This only returns on error
```

Comment:
Should move this comment down to the `dn.finalizeAndReboot` now I suppose?

Reply:
umm, we will return back here also when updateOS() fails?